docs/guides/tool-discovery.md

```
@@ -0,0 +1,185 @@
+
+This creates a Kind cluster, installs Istio as the Gateway API provider, deploys the MCP Gateway controller and broker, and registers the default test MCP servers.
+
+Verify the gateway is running:
+
+```bash
+kubectl -n mcp-system get deployment mcp-gateway
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)